src/pages/api/sepana/delete.page.ts

```diff
@@ -0,0 +1,45 @@
+import { NextApiHandler } from 'next'
+
+import { deleteSepanaDoc, sepanaAlert } from './utils'
+
+const handler: NextApiHandler = async (req, res) => {
+  const { id } = req.query
+
+  if (typeof id !== 'string' || !id.length) {
+    res.status(400).end()
+    return
+  }
+
+  try {
+    await deleteSepanaDoc(id).then(async _res => {
+      await sepanaAlert({
+        type: 'alert',
+        alert: 'DELETED_RECORD',
+        body: `ID: ${id}`,
+      })
+
+      res.status(200).json(_res.data)
+    })
```

Comment:
**suggestion** think we can avoid nested `then` here

```suggestion
    const _res = await deleteSepanaDoc(id)
    await sepanaAlert({
      type: 'alert',
      alert: 'DELETED_RECORD',
      body: `ID: ${id}`,
    })

    return res.status(200).json(_res.data)
    })
```

Reply:
bet